bot/services/inspiro_client.go

```diff
@@ -57,12 +58,13 @@ func NewInspiroClientWithConfig(myConfig InspiroConfig) *InspiroClient {
 	}
 }
 
-// Abstracted 'Constructor'
+// NewInspiroClient constructs an InspiroClient with the default configuration.
```

Comment:
Well, no actually. It constructs an InspiroClient with the config returned by the default Getter. This isn't the default configuration as it's still using the application config. We don't know which values are default and which are user provided. This is effectively a wrapper function for default behavior not the default configuration. @jules